src/compiler/factory.ts

```diff
@@ -1032,17 +1032,19 @@ namespace ts {
             : node;
     }
 
-    export function createTaggedTemplate(tag: Expression, template: TemplateLiteral) {
+    export function createTaggedTemplate(tag: Expression, typeArguments: NodeArray<TypeNode>, template: TemplateLiteral) {

```

Comment:
the API would be easier to use if the parameter is typed as `ReadonlyArray` and it's converted to a `NodeArray` in the function body.
this also applies to the update function below.

Reply:
Good call!